src/autosuggest/index.tsx

```diff
@@ -25,6 +28,15 @@ const Autosuggest = React.forwardRef(
         virtualScroll: props.virtualScroll,
       },
     });
+
+    const componentAnalyticsMetadata: GeneratedAnalyticsMetadataAutosuggestComponent = {
+      name: 'awsui.Autosuggest',
+      label: 'input',
+      properties: {
+        disabled: `${!!props.disabled}`,
```

Comment:
Should this also include readonly?

Reply:
I can include it at a later stage if needed